doc/api/crypto.md

```diff
@@ -1365,6 +1365,13 @@ The `key` is the raw key used by the `algorithm` and `iv` is an
 [Buffers][`Buffer`], `TypedArray`, or `DataView`s. If the cipher does not need
 an initialization vector, `iv` may be `null`.
 
+Initialization vectors should be unpredictable; ideally, they will be
```

Comment:
This really depends on the cipher block mode, some work fine even with predictable IVs. What is much more important is that IVs are unique and that their value cannot be influenced by an attacker.

Reply:
You're right. How should I better communicate this, in your opinion?

I (among a lot of other people, I'm sure), prefer to keep the IV unpredictable (cryptographically random) anyway. Better be safe than be sorry, I guess.

Reply:
Maybe `should be unpredictable and unique`? That's not always true, but as you said, better safe than sorry.

Reply:
Sure, I'll add `unique`. I think that was exactly what @bnoordhuis wanted to communicate as well.